playbooks/roles/ocp-service-binding-operator/tasks/main.yml

```diff
@@ -48,6 +48,14 @@
 - name: Check all nodes are healthy
   shell: oc wait --all --for=condition=Ready nodes 
 
+# Get the cluster version
+- name: Get the cluster version
+  shell: oc version | grep -i server | cut -d':' -f2 | cut -d'.' -f1-2
```

Comment:
Instead of `-i` in grep it is better to check for `Server Version` so we are sure what we are checking for.

Reply:
ok, will check